athena-vertica/sar_bucket_policy.json

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
I'm not sure this file should be comitted, isn't this a generated file?

Similarly, should athena-vertica/av.jpg  be comitted or was that also a mistake?

Reply:
Will remove the sar_bucket_policy.json file. 
av.jpg refers to the connector architecture diagram in the ReadMe file. I think we should keep this as this is the first connector where we Export the data to S3 instead of getting it back through the JDBC bridge.